linux_docker_resources/Dockerfile-RHEL

```diff
@@ -119,8 +120,21 @@ RUN dnf install \
     --refresh -y
 
 # Get and install the RTI web binaries.
-RUN cd /tmp && curl --silent https://s3.amazonaws.com/RTI/Bundles/5.3.1/Evaluation/rti_connext_dds_secure-5.3.1-eval-x64Linux3gcc5.4.0.tar.gz | tar -xz
-RUN cd /tmp && tar -xvf /tmp/openssl-1.0.2n-target-x64Linux3gcc5.4.0.tar.gz
+# Connext 5.3.1 for galactic and earlier.
+RUN if [ "$ROS_DISTRO" = "foxy" -o "$ROS_DISTRO" = "galactic" ]; then \
+       cd /tmp && curl --silent https://s3.amazonaws.com/RTI/Bundles/5.3.1/Evaluation/rti_connext_dds_secure-5.3.1-eval-x64Linux3gcc5.4.0.tar.gz | tar -xz \
+       && tar -xvf /tmp/openssl-1.0.2n-target-x64Linux3gcc5.4.0.tar.gz; \
```

Comment:
It made more sense to me to make this one long multiline RUN than it did to perform the same check on two consecutive RUN lines.

Reply:
Agreed with that, but isn't this missing a `fi` somewhere?

Reply:
sigh. lost in the rebases.

Reply:
[9bc507e](https://github.com/ros2/ci/pull/652/commits/9bc507e1ebfdd2b78b79599b880b9f067779d274)